README.md

```diff
@@ -23,3 +23,5 @@ const wayscript = require("wayscript")
 event = wayscript.context.getEvent();
 console.log(event);
 ```
+
+More use cases in example.js
```

Comment:
Could we make this a relative link?

I think the syntax should be something like:

"More use cases in can be found [here](example.js)."

based on what I found here: https://github.blog/2013-01-31-relative-links-in-markup-files/